test/lit/binary/dwarf-multivalue.test

```diff
@@ -97,9 +97,8 @@
 ;; ROUNDTRIP-NEXT:  (local $11 i32)
 ;; ROUNDTRIP-NEXT:  (local $12 f32)
 ;; ROUNDTRIP-NEXT:  (local $13 i32)
-;; ROUNDTRIP-NEXT:  (local $14 f32)
-;; ROUNDTRIP-NEXT:  (local $15 (tuple i32 f32))
-;; ROUNDTRIP-NEXT:  (local $16 i32)
+;; ROUNDTRIP-NEXT:  (local $14 (tuple i32 f32))
+;; ROUNDTRIP-NEXT:  (local $15 i32)
```

Comment:
The comment on line 80 looks like it needs to be updated.

Reply:
Yes, good point, I think we no longer emit `(local $14)`. I updated the comments.